spec/v1/bundle.md

```diff
@@ -2,40 +2,63 @@
 
 Version: v1.0
 
-An attestation bundle is a collection of multiple attestations in a single
+An attestation Bundle is a collection of multiple attestations in a single
 file. This allows attestations from multiple different points in the software
 supply chain (e.g. Provenance, Code Review, Test Result, vuln scan, ...) to
 be grouped together, allowing users to make decisions based on all available
 information.
 
-**NOTE**: The bundle is not authenticated as a whole. Instead each individual
-attestation is authenticated using [DSSE]. As such, an attacker might be able
-to _delete_ an attestation without being detected. Predicates that follow
-[the monotonic principle] should not have any issues with this behavior.
+**NOTE**: The Bundle is not authenticated as a whole. Instead each individual
+attestation is authenticated using signature schemes like [DSSE]. As such,
+an attacker might be able to _delete_ an attestation without being detected.
+Predicates that follow [the monotonic principle] should not have any issues
+with this behavior.
 
 ## Data structure
 
 Attestation Bundles use [JSON Lines] to store multiple attestations.
 
--   Each attestation within a bundle MAY have a different signing key,
+-   Each attestation within a Bundle MAY have a different signing key,
     `_type`, `subject`, and/or `predicateType`.
--   Each line SHOULD be an [Envelope]. Consumers MUST ignore unrecognized
-    lines.
+-   Each line within a Bundle SHOULD be an [Envelope].
+-   Each [Envelope] within a Bundle MAY use a different signature format,
+    but it MUST fit on a single JSON line.
+-   Consumers MUST ignore unrecognized lines.
 -   Consumers MUST ignore attestations with unrecognized keys, types,
     subjects, or predicates.
--   Processing of a bundle MUST NOT depend on the order of the attestations.
+-   Processing of a Bundle MUST NOT depend on the order of the attestations.
 
 ## File naming convention
 
 Bundles SHOULD use the suffix `.intoto.jsonl`.
 
-A bundle of attestations relevant for `<filename>` SHOULD be named
-`<filename>.intoto.jsonl`. Attestations in the bundle MAY have different
-subjects, but they SHOULD all be relevant to that file or its dependencies.
-For example, a package named `foo-1.2.3.tar.gz` with hash `abcd` that was
-built from git commit `1234` could have a bundle name
-`foo-1.2.3.tar.gz.intoto.jsonl` with two attestations, one with subject
-`abcd` and one with subject `1234`.
+<!-- @marcelamelara: This likely needs to be updated for ITE-10/11 -->
```

Comment:
Can we file an issue for this?

Reply:
We probably should. The reason I hadn't was because this comment is based on how the [ITE-10/11 prototype](https://github.com/adityasaky/in-toto-attestation-verifier) that @adityasaky built currently works. So, I don't know if the Bundle/Envelope file naming expectations are specific to the prototype or if they will eventually become part of the spec. To my recollection, ITE-10/11 don't currently include conventions for file naming for verification purposes.

Reply:
The issue in ITE-10 is less with the name of the bundle as a whole but rather with mapping individual attestations to the steps they correspond to. The bundle format is ill suited to associating attestations with their steps. in-toto 1.0 leans on storing each link metadata on disk using the step name, and for good measure, also includes the step name in the predicate.

Reply:
Gotcha. This is probably something we ought to clarify at the Envelope layer then, to also ensure we're in spec with in-toto 1.0.

Reply:
@adityasaky @joshuagl if you could PTAL at the updated text!